locales/en/dashboard.json

```diff
@@ -2,24 +2,28 @@
   "page_title": "Your overview",
   "sections": {
     "outgoing": {
-      "heading": "$t(primary_navigation.outgoing) for today",
+      "heading": "$t(primary_navigation.outgoing) for",
       "summary": {
         "total": "outgoing move",
         "total_plural": "outgoing moves"
       }
     },
     "incoming": {
-      "heading": "$t(primary_navigation.incoming) for today",
+      "heading": "$t(primary_navigation.incoming) for",
```

Comment:
I wonder if it would be better to pass the date into the heading. Otherwise a screen reader would just read out “heading one, incoming moves for”.

Reply:
Refactored. I propagated the change to the rest of the dashboards since we had the same issue everywhere else.